tests/components/modbus/test_modbus_climate.py

```diff
@@ -0,0 +1,47 @@
+"""The tests for the Modbus cover component."""
```

Comment:
Please fix the typo. Not cover, but climate component.

Reply:
Guilty as charged, that happens when you copy files :-) I have fixed it in an additional commit.